src/components/editions/article.tsx

```diff
@@ -0,0 +1,62 @@
+// ----- Imports ----- //
+
+import React, { FC } from 'react';
+import { css } from '@emotion/core';
+import { border } from '@guardian/src-foundations/palette';
+import { remSpace } from '@guardian/src-foundations';
+import { Lines } from '@guardian/src-ed-lines';
+import { Design } from '@guardian/types/Format';
+import { partition } from '@guardian/types/result';
+
+import { Item } from 'item';
+import { renderAll } from 'renderer';
+import Standfirst from 'components/editions/standfirst';
+import HeaderImage from 'components/editions/headerImage';
+import Series from 'components/editions/series';
+import Headline from 'components/editions/headline';
+import Byline from 'components/editions/byline';
+
+
+// ----- Component ----- //
+
+interface Props {
+    item: Item;
+}
+
+const headerStyles = css`
+    margin: 0 ${remSpace[3]} ${remSpace[4]};
+`;
+
+const bodyStyles = css`
+    border-top: 1px solid ${border.secondary};
+    padding: 0 ${remSpace[4]};
+`;
+
+const Article: FC<Props> = ({ item }) => {
+    if (item.design === Design.Live) {
+        return <p>Not implemented</p>
+    }
+
+    return (
+        <main>
+            <article>
+                <header css={headerStyles}>
```

Comment:
Wouldn't it be better to use `emotion.styled` as you already seperate out the css and turn it into a component? More stylistic than anything

Reply:
We could do. Bear in mind that the `css` prop is used everywhere else in this codebase so it would be a divergence from that, but also happy to consider switching if you think there are advantages to `styled`. What do you reckon it gives us over the `css` prop?